src/styles/main.scss

```diff
@@ -1,3 +1,79 @@
-body {
+$calendar-border: 20px 10px;
+$calendar-gaps: 1px;
+$day-size: 100px;
+$day-background: #eee;
+$day-background-hover: pink;
+$day-border-color: #000;
+$day-font: arial, helvetica, sans-serif;
```

Comment:
Lector Told us today that most projects will have all/most things like colors, fonts and sizes put into variable even if they are not reused. Additionally, once I start putting variables into a separate SCSS file, it will be easier to change them rather than delve into the file they are important for.